default_settings.py

```diff
@@ -87,7 +87,7 @@
 
 # Context layers. The order of the layers that we want to show
 # in the map report.
-LAYER_ORDER = [
+REPORT_LAYER_ORDER = [
```

Comment:
@myarjunar I proposed we change the setting's name into this, so we know this is related with report and not map client.